### ISSUE TITLE
[Accessibility] Layer5 logo link does not have a discernable name

### ISSUE THREAD

**Issue by OMGATE23** [layer5io/layer5#4265]
### Description
This is an accessibility issue. The Layer5 logo is a link and is an SVG wrapped around an 'a' tag. Due to this, it does not have any name or label which can tell screen readers or other accessibility tools to help identify the purpose of the link or what the link stands for.
The logo can be seen inside the header on our home page here: https://layer5.io/

I would like to work on this.

### Expected Behavior
The 'a' tag can have an aria-label reading 'layer5' to avoid this issue.  

### Screenshots

Following is the logo element in header:

![image](https://github.com/layer5io/layer5/assets/90276220/1f8e9277-b453-42de-98fb-cb8dc40ca4f2)

 Following is what lighthouse had to say about this accessibility issue:
 
![layer5 link discernable link](https://github.com/layer5io/layer5/assets/90276220/c3e9506d-16dc-468e-8ab8-3fe6ddd4229a)

### Environment:
- Host OS: Windows
- Browser: Brave

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.png" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by OMGATE23** [comment]
I would like to work on this issue

**Comment by Savio629** [comment]
Assigned to you @OMGATE23
